pom.xml

```diff
@@ -56,22 +56,25 @@
 		</repository>
 	</repositories>
 	<properties>
+		<maven.compiler.source>1.8</maven.compiler.source>
```

Comment:
Can drop this down if desired, but I think nowadays this is a reasonable baseline